guacamole/src/main/frontend/src/app/client/types/ManagedFilesystem.js

```diff
@@ -335,6 +348,72 @@ angular.module('client').factory('ManagedFilesystem', ['$rootScope', '$injector'
 
     };
 
+    /**
+     * translate the permission number to the permission string
+     *
+     * @param {Number} permission
+     */
+    ManagedFilesystem.permissionTranslate = function permissionTranslate(permission) {
+
+        var Ow_R = 256;
+        var Ow_W = 128;
+        var Ow_X = 64;
+        var Gp_R = 32;
+        var Gp_W = 16;
+        var Gp_X = 8;
+        var Ot_R = 4;
+        var Ot_W = 2;
+        var Ot_X = 1;
```

Comment:
1. These should be defined (and documented) as constants: https://guacamole.apache.org/guac-style/#comments-and-documentation
2. These should be named according to constant naming convention (`UPPERCASE_WITH_UNDERSCORES`): https://guacamole.apache.org/guac-style/#naming